stl/src/filesys.cpp

```diff
@@ -15,6 +15,9 @@
 
 #include <Windows.h>
 
+extern "C" DWORD __cdecl __crtGetTempPath2W(

```

Comment:
If we're going to replicate the SAL annotations, it seems that we should also replicate the return annotation, which is `_Success_(return > 0 && return < BufferLength)`. Also applies to the definition, and to `_Stl_GetTempPath2W()` in `filesystem.cpp`.

There are no current examples, but there was a previous example: https://github.com/microsoft/STL/blob/5bf2826707a201290562dcb5bb8c63e433784737/stl/src/awint.hpp#L162-L164